pkg/ceph/client/pool.go

```diff
@@ -124,6 +137,32 @@ func CreatePoolWithProfile(context *clusterd.Context, clusterName string, newPoo
 	return CreatePoolForApp(context, clusterName, newPool, appName)
 }
 
+func DeletePool(context *clusterd.Context, clusterName string, name string) error {
+	// check if the pool exists
+	pool, err := GetPoolDetails(context, clusterName, name)
+	if err != nil {
+		logger.Infof("pool %s not found for deletion. %+v", name, err)
+		return nil
+	}
+
+	logger.Infof("purging pool %s (id=%d)", name, pool.Number)
+	args := []string{"osd", "pool", "delete", name, name, reallyConfirmFlag}
```

Comment:
confirmFlag is not enough?

Reply:
correct, you have to pass the name twice and the really confirm flag. I'm telling you, ceph makes it hard to delete pools :)